allure-generator/webpack/webpack.dev.js

```diff
@@ -0,0 +1,68 @@
+const webpack = require("webpack");
+const writeFilePlugin = require("write-file-webpack-plugin");
+const webpackMerge = require("webpack-merge");
+const FriendlyErrorsWebpackPlugin = require("friendly-errors-webpack-plugin");
+const SimpleProgressWebpackPlugin = require("simple-progress-webpack-plugin");
+const WebpackNotifierPlugin = require("webpack-notifier");
+const sass = require("sass");
+const utils = require("./utils.js");
+
+const commonConfig = require("./webpack.common.js");
+
+const ENV = "development";
+
+const postcssLoader = {
+  loader: "postcss-loader",
+  options: {
+    plugins: [require("autoprefixer")({ browsers: ["last 10 versions"] })]
+  }
+};
+
+module.exports = options =>
+  webpackMerge(commonConfig({ env: ENV }), {
+    devtool: "cheap-module-source-map", // https://reactjs.org/docs/cross-origin-errors.html
+    mode: ENV,
+    entry: ["./src/main/javascript/index.js"],
+    output: {
+      path: utils.root("build/www"),
+      publicPath: "/",
+      filename: "app.js"
+    },
+    module: {
+      rules: [
+        {
+          test: /\.(sa|sc|c)ss$/,
```

Comment:
You have only `.scss` files, right?
Let's make it `/\.scss$/` for the simplicity.